client/src/components/Lobby.svelte

```diff
@@ -15,6 +15,10 @@
   export let lobby: Lobby;
   let errorMessage: string = "";
 
+  // variable that will be set true if the corresponding function has no errors thrown
+  // this will then allow the button to be pressed again if there is an error thrown
+  let once: boolean = false;
```

Comment:
I think we could find a more descriptive variable name maybe `startingGame` or `waiting` or something like that